BridgeAppSDK/SBATrackedDataObjectCollection.m

```diff
@@ -54,16 +54,23 @@ - (NSString *)schemaIdentifier {
 
 - (SBATrackedDataStore *)dataStore {
     if (_dataStore == nil) {
-        _dataStore = [SBATrackedDataStore defaultStore];
+        _dataStore = [SBATrackedDataStore sharedStore];
     }
     return _dataStore;
 }
 
-- (NSNumber *)repeatTimeInterval {
-    if (_repeatTimeInterval == nil) {
-        _repeatTimeInterval = @(30 * 24 * 60 * 60);   // Every 30 days by default
+- (NSNumber *)trackingSurveyRepeatTimeInterval {
+    if (_trackingSurveyRepeatTimeInterval == nil) {
+        _trackingSurveyRepeatTimeInterval = @(30 * 24 * 60 * 60);   // Every 30 days by default
```

Comment:
This is approximate time interval. As in "about once a month, query the user for any changes to their answers from the last time we asked". I wanted to use a time interval here because it is something that can be set in a json file and is *not* number of days (in case there is a reason for it to be less that 1 day.